chef/site-cookbooks/wca/templates/env.production.erb

```diff
@@ -14,3 +14,4 @@ SMTP_PASSWORD=<%= @secrets['SMTP_PASSWORD'] %>
 RECAPTCHA_PUBLIC_KEY=<%= @secrets['RECAPTCHA_PUBLIC_KEY'] %>
 RECAPTCHA_PRIVATE_KEY=<%= @secrets['RECAPTCHA_PRIVATE_KEY'] %>
 NEW_RELIC_LICENSE_KEY=<%= @secrets['NEW_RELIC_LICENSE_KEY'] %>
+GOOGLE_MAPS_API_KEY=<%= "AIzaSyA-kN3GCsG78MnAp16CIRft221XNmioovk" %>
```

Comment:
We don't have to care as it comes with the html code (is included in application.html.erb).